bindings/java/src/main/native/Makefile.am

```diff
@@ -22,6 +22,9 @@ DISTCLEANFILES = org_ucx_jucx_ucp_UcpConstants.h \
                  org_ucx_jucx_ucs_UcsConstants.h \
                  org_ucx_jucx_ucp_UcpListener.h
 
+MOSTLYCLEANFILES = $(topdir)/bindings/java/src/main/native/build-java \
+                   $(topdir)/.deps
```

Comment:
why need to add .deps? aren't they already cleaned by Automake?
also, it wasn't in the 'rm -rf' version of this PR..

Reply:
It doesn't cleaned by automake.

Reply:
We **may** or **may not** clean that folder.  Maybe better to remove it, if we want to transfer ucx folder as a clean source. `.deps` folder contains downloaded dependencies, that would take space

Reply:
should be part of distclean. i don't think we should change automake logic here
also, can we replace `$(topdir)/bindings/java/src/main/native/` with `$(builddir)` ?
Makefile:
```
distclean: distclean-am
        -rm -rf ./$(DEPDIR)
        -rm -f Makefile
```

Reply:
```test -z "./build-java" || rm -f ./build-java
rm: cannot remove ‘./build-java’: Is a directory
```
https://www.gnu.org/savannah-checkouts/gnu/automake/manual/html_node/Clean.html: 
```
A common case is deleting a directory, for instance, a directory created by the test suite:

clean-local:
        -rm -rf testSubDir
```

Reply:
so back to clean-local ?